Britannia/localization/english/BTWK_storyline_events_l_english.yml

```diff
@@ -1,36 +1,72 @@
+ BTWK_story_event.0113.b:0 "As much as it pains me, he is right."
+
+ BTWK_story_event.0104.t:0 "Choosing Mordred's Fosterer"
+ BTWK_story_event.0104.desc:0 "The most pressing matter of discussion is finding Mordred a foster father. The warlord you choose to appoint to the role of Mordred's foremost guardian will in return benefit from a strong alliance with your kingdom. Only those who are unmarried, or rulers with unmarried heirs, are eligible however. In order to seal the pact, you plan to marry the chosen candidate to your daughter-in-law, the late Mordred's widow, and the younger Mordred's mother - Lady Norwenna.\n\n After much debate, the list of suitors has been whittled down to a small selection. Who will you choose?"
+ BTWK_story_event.0104.a:0 "[powys_suitor.GetShortUIName] of [GetTitleByKey('k_powys').GetNameNoTier]"
+ BTWK_story_event.0104.b:0 "[siluria_suitor.GetShortUIName] of [GetTitleByKey('k_siluria').GetNameNoTier]"
+ BTWK_story_event.0104.c:0 "[kernow_suitor.GetShortUIName] of [GetTitleByKey('k_kernow').GetNameNoTier]"
+ BTWK_story_event.0104.d:0 "[gwent_suitor.GetShortUIName] of [GetTitleByKey('k_gwent').GetNameNoTier]"
+ BTWK_story_event.0104.e:0 "[glevum_suitor.GetShortUIName] of [GetTitleByKey('k_glevum').GetNameNoTier]"
+ BTWK_story_event.0104.f:0 "[belgae_suitor.GetShortUIName] of [GetTitleByKey('k_belgae').GetNameNoTier]"
+ BTWK_story_event.0104.g:0 "[cynwidion_suitor.GetShortUIName] of [GetTitleByKey('k_cynwidion').GetNameNoTier]"
+ BTWK_story_event.0104.h:0 "[demetia_suitor.GetShortUIName] of [GetTitleByKey('k_demetia').GetNameNoTier]"
+ BTWK_story_event.0104.i:0 "[prince_arthur.GetFirstName] is still an option..."
+
+ BTWK_story_event.0105.t:0 "Mordred's Guardians"
+ BTWK_story_event.0105.desc:0 "DESC"
  BTWK_story_event.0105.a:0 "Send word to Arthur, though it grieves me greatly..."
 
+ BTWK_story_event.0114.t:0 "The Council's Conclusion"
+ BTWK_story_event.0114.a:0 "To us!"  # needs reloc to something else
+ BTWK_story_event.0114.b:0 "It looks like we will have to forge our own path."
+ BTWK_story_event.0114.c:0 "I pray we have done enough to stave off the invasions." 
+ BTWK_story_event.0114.d:0 "May these alliances prove fruitful, for all our sakes."
+
+ BTWK_story_event.0115.t:0 "A Summit in Glevum"
+ BTWK_story_event.0115.desc_header:0 "The High King of Britannia, Uther Pendragon, has recently hosted a High Council in the villa of Glevum."
+ BTWK_story_event.0115.desc_alliance:0 "Word has reached you that through arranging a marriage between Norwenna - the widow of the late Prince of Dumnonia, Mordred - and (SPOUSE), Uther has secured an alliance between his realm, and (NEW REALM). A number of guardians have also been named for the Edling Mordred. Chiefly among them is Tewdric, King of Gwent, the legendary druid Merlin, and Uther's own son, the renowned warlord Prince Arthur. Such a web of alliances will surely fortify Dumnonia against its enemies until the young Prince reaches maturity."
```

Comment:
- I would rephrase the first sentence to focus more on the alliance and then more mentioning the marriage in passing
- "widow of the late Prince of Dumnonia" -> "widow of the late heir of Dumnonia"
- "Chiefly among" -> "Chief among"
- "reaches maturity" -> "comes of age" ?